src/clarity-angular/select/_select.clarity.scss

```diff
@@ -1,25 +1,26 @@
 @include exports('select.clarity') {
-    clr-select clr-select-menu ul {
-        list-style-type: none   
+    .clr-select .clr-select-menu ul {
+        list-style-type: none
     }
-    clr-option  li:hover, li.highlight {
-        background: #eee
+    .clr-option {
+        display: list-item;
+        text-align: -webkit-match-parent;
```

Comment:
Is there an equivalent we should have for other browsers? Did you test the alignment on IE, and Firefox?